draft-private-access-tokens.md

```diff
@@ -477,31 +556,45 @@ accept = message/access-token-response
 cache-control = no-cache, no-store
 content-type = message/access-token-request
 content-length = 512
-Anonymous-Origin-ID = ANON_ORIGIN_ID
+anonymous-origin-id = ANON_ORIGIN_ID
+client-id = CLIENT_ID
+request-nonce = request_nonce
 
 <Bytes containing the AccessTokenRequest>
 ~~~
 
-Upon receipt, the Mediator computes ANON_CLIENT_ID, which is a fixed-length byte string
-for the given Client. See {{client-id}} for details of its computation. The Mediator
-also computes ANON_ORIGIN_ID_PRIME, a fixed-length byte string, for each ANON_ORIGIN_ID
-for a specific ANON_CLIENT_ID. See {{origin-id}} for details its computation.
+If the Mediator detects a version in the AccessTokenRequest that it does not recognize
+or support, it MUST reject the request with an HTTP 400 error.
 
 The Mediator also checks to validate that the key_id in the client's AccessTokenRequest
-matches a known ISSUER_NAME_KEY public key for the Issuer. For example, the Mediator can
+matches a known ISSUER_KEY public key for the Issuer. For example, the Mediator can
 fetch this key using the API defined in {{setup}}. This check is done to help ensure that
 the Client has not been given a unique key that could allow the Issuer to fingerprint or target
 the Client. If the key does not match, the Mediator rejects the request with an HTTP
 400 error. Note that Mediators need to be careful in cases of key rotation; see
 {{privacy-considerations}}.
 
-If the Mediator detects a version in the AccessTokenRequest that it does not recognize
-or support, it MUST reject the request with an HTTP 400 error.
+The Mediator finally checks to ensure that the AccessTokenRequest.request_tag value
+matches its own calculation of request_tag = HashToGroup(CLIENT_ID) * request_nonce.
+If the request_tag does not match, the Mediator rejects the request with an HTTP
+400 error.
+
+TODO: Chris to update this
+
+If the Mediator accepts the request, it will look up the state stored for this Client.
```

Comment:
AFAIU, its a little odd that the mediator needs to look up the Client based on a "client ID" it has for the client (device identity, IP, etc), which is different from the new "CLIENT_ID" that the client has provided?

Reply:
Yes, it needs its own notion of the client...